### PR TITLE
Assorted Improvements

### PR DESCRIPTION
**Pull request by mdiluz**
Refactor the README now that GameMode is more established - see commits for full list of changes. See [my branch's page](https://github.com/mdiluz/gamemode/tree/minor-improvements) for what the end result looks like. Sorry for the big diff!

Also removes logging an error by default about the renice value.

### CODE REVIEW

File: daemon/gamemode-sched.c [hunk 1]
```diff
@@ -74,11 +69,7 @@ void game_mode_apply_renice(const GameModeContext *self, const pid_t client)
 	long int renice = 0;
 	config_get_renice_value(config, &renice);
 	if ((renice < 1) || (renice > 20)) {
-		LOG_ONCE(ERROR,
-		         "Invalid renice value '%ld' reset to default: %d.\n",
-		         renice,
-		         -NICE_DEFAULT_PRIORITY);
-		renice = NICE_DEFAULT_PRIORITY;
+		LOG_ONCE(ERROR, "Invalid renice value '%ld' is invalid, will not renice.\n", renice);
```
Comment (aejsmith) [inline daemon/gamemode-sched.c]: "Invalid ... is invalid"

Comment (mdiluz) [reply daemon/gamemode-sched.c]: Haha, touche, I'll fix it up!

Comment (mdiluz) [reply daemon/gamemode-sched.c]: 66bd6a3 fixes this horrible error. Also makes it actually bail out, as it really should.

**Comment by mdiluz** [comment]
(Alex is away, so updating the PR with some other small things before he gets back).

* Added -t to the man page
* Ensured inhibit_screensaver was configurable

**Comment by mdiluz** [comment]
* Prevent double printouts in the log from cpugovctl
* Add `gamemoderun` (behaves just like `primusrun`)

**Comment by mdiluz** [comment]
* Add a note about Ubuntu Cosmic package (woo!)
* Actually have gamemode use the error channel for error logs, and prepend those logs with `ERROR: `

**Comment by kakra** [comment]
LGTM... And I like the run-wrapper, this should prevent the many problems people seem to have with pasting `\$LIB` correctly.

**Comment by mdiluz** [comment]
* Fix the tests sometimes failing on null memory

**Comment by mdiluz** [comment]
Derp, I'll write a test for it :)

**Pull request merged**